pkgs/applications/misc/opentrack/default.nix

```diff
@@ -36,18 +37,26 @@ in
       owner = "opentrack";
       repo = "opentrack";
       rev = "opentrack-${version}";
-      sha256 = "sha256-8gpNORTJclYUYp57Vw/0YO3XC9Idurt0a79fhqx0+mo=";
+      sha256 = "sha256-C0jLS55DcLJh/e5yM8kLG7fhhKvBNllv5HkfCWRIfc4=";
     };
 
-    nativeBuildInputs = [cmake pkg-config ninja copyDesktopItems];
+    patches = [
+      # https://github.com/opentrack/opentrack/pull/1754
+      ./0001-fix-wine-no-wrapper.patch
```

Comment:
can we use fetchpatch here instead ?

url is something like https://github.com/opentrack/opentrack/commit/d501d7e0b237ed0c305525788b423d842ffa356d.patch

Reply:
Done.